### PR TITLE
Add event logging to signup Cody toast and email verification toast

### PR DESCRIPTION
**Pull request by dadlerj**
## Test plan

- sign up for a new account on dotcom

### CODE REVIEW

**Review by erzhtor — APPROVED** [review]
Pushed some minor changes.

File: client/web/src/marketing/toast/CodySurveyToast.tsx [hunk 1]
```diff
@@ -45,13 +49,19 @@ const CodySurveyToastInner: React.FC<{ onSubmitEnd: () => void }> = ({ onSubmitE
 
     const handleSubmit = useCallback(
```
Comment (thenamankumar) [inline client/web/src/marketing/toast/CodySurveyToast.tsx:50]: We can remove the useCallback if this is dependent on the state.

**Comment by robandmaggierhyne** [comment]
Thanks for adding these events!

**Comment by dadlerj** [comment]
@thenamankumar sorry, I'm no expert on react... Should I make that change before merging, or is it an optional/stylistic thing?

**Pull request merged**